doc/content/getting_started/installation.md

```diff
@@ -59,20 +59,43 @@ To compile FENIX, first make sure that the conda MOOSE environment is activated
 conda activate moose
 ```
 
-Then navigate to the FENIX clone directory and download the MOOSE submodule:
+Then navigate to the FENIX clone directory and download the MOOSE and TMAP8 submodules:
 
 ```bash
 cd ~/projects/FENIX
 git submodule update --init moose
+git submodule update --init tmap8
 ```
 
-!alert note
-The copy of MOOSE provided with FENIX has been fully tested against the current
+Finally, Cardinal and its dependencies must be downloaded and pre-built. To perform this, first run
+the build script from the FENIX clone directory:
+
+```bash
+scripts/update_and_rebuild_cardinal.sh
+```
+
+Then, to download OpenMC cross sections needed for OpenMC-based Cardinal runs, run:
+
+```bash
+cd ~/projects/FENIX/cardinal
+scripts/download-openmc-cross-sections.sh
+```
+
+and subsequently set the location of those cross sections in your environment:
+
+```bash
+export OPENMC_CROSS_SECTIONS=~/projects/FENIX/cross_sections/cross_sections.xml
```

Comment:
Reminder to add a warning to the documentation here letting the user know that this must be set in the environment *every time* the Terminal is reloaded (unless it is set persistently in the shell profile).

Reply:
Are you going to add this reminder in this PR?

Reply:
Yes - I like to avoid too many changes after I've requested reviews, and then integrate my comments with the review changes. :)

Reply:
Addressed in 2927d33